incubator/minecraft/templates/deployment.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.Image }}:{{ .Values.ImageTag }}"
+        imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: "{{.Values.Cpu}}"
+            memory: "{{.Values.Memory}}"
+          limits:
+            cpu: "{{.Values.Cpu}}"
```

Comment:
So I would perfer to set both separately. You set mem in redis to the same value, because of the app. You don't want k8s sneaking mem from it... 

But I am not a minecraft guru.